openreview/conference/templates/paper_recruitment_pre_process.py

```diff
@@ -0,0 +1,26 @@
+def process(client, note, invitation):
+    from Crypto.Hash import HMAC, SHA256
+    import urllib.parse
+    HASH_SEED = ''
+    REVIEWERS_INVITED_ID = ''
+    REVIEWERS_REGEX = ''
+    CHECK_DECLINE = False
+
+    user = urllib.parse.unquote(note.content['user'])
+    hashkey = HMAC.new(HASH_SEED.encode(), digestmod=SHA256).update(user.encode()).hexdigest()
+
+    submission = client.get_notes(note.content['submission_id'], details='original')[0]
+
+    if hashkey != note.content['key']:
+        raise openreview.OpenReviewException('Wrong key, please refer back to the recruitment email')
+
+    if not client.get_groups(id=REVIEWERS_INVITED_ID, member=user):
+        raise openreview.OpenReviewException('User not in invited group, please accept the invitation using the email address you were invited with')
+
+    if 'Desk_Rejected_Submission' in submission.invitation or 'Withdrawn_Submission' in submission.invitation:
+        raise openreview.OpenReviewException('This submission is no longer under consideration. Please contact the paper area chair or program chairs to be unassigned.')
```

Comment:
@celestemartinez could you help here with the error message? The user that accept or declines an invitation to review a paper that is not under review anymore.

Reply:
What about
> This submission is no longer under review. No action is required from your end. 

or something similar? I don't think the AC/PCs should be contacted to unassign them (if I am not mistaken, the edge browser won't even show this paper anymore).

Reply:
That's true, the user should ignore the invitation.